src/config/acbs.config.ts

```diff
@@ -0,0 +1,36 @@
+import { registerAs } from '@nestjs/config';
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+export interface AcbsConfig {
+  apiKey: string;
+  apiKeyHeaderName: string;
+
+  authentication: {
+    baseUrl: string;
+    clientId: string;
+    loginName: string;
+    password: string;
+  };
+
+  maxRedirects: number;
+  timeout: number;
+}
+
+export default registerAs(
+  'acbs',
+  (): AcbsConfig => ({
+    apiKey: process.env.ACBS_API_KEY,
+    apiKeyHeaderName: process.env.ACBS_API_KEY_HEADER_NAME,
+
+    authentication: {
+      baseUrl: process.env.ACBS_AUTHENTICATION_BASE_URL,
+      clientId: process.env.ACBS_AUTHENTICATION_CLIENT_ID,
+      loginName: process.env.ACBS_AUTHENTICATION_LOGIN_NAME,
+      password: process.env.ACBS_AUTHENTICATION_PASSWORD,
+    },
+
+    maxRedirects: parseInt(process.env.ACBS_MAX_REDIRECTS) || 21,
```

Comment:
21 is the default value of maxRedirects in Axios

Reply:
do we expect to have that many?

Reply:
We don't have documentation for the endpoints we use to get the token

The existing API _does_ follow redirects, and we think it's using the default number of max redirects for Mulesoft (which is 5 according to [this article](https://help.mulesoft.com/s/article/How-to-increase-number-of-redirects-when-you-get-MaxRedirectException-in-HTTPClient)). @avaitonis says they haven't had issues with the redirect limit being exceeded in the past.

I'll reduce our default down to 5 as well

Reply:
sounds good